modules/ComscoreStreamingTag/ComscoreStreamingTag.loader.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * ComscoreStreamingTag loader
+ */
+( function( mw ) { "use strict";
+	mw.addKalturaPlugin( ["ComScoreStreamingTag"], 'ComScoreStreamingTag', function( embedPlayer, callback ){
```

Comment:
not your fault but our convention is a bit odd, we have uppercase module names in the directory and lowercase first letter camel case names for plugins. For example "doubleClick" or "nielsenCombined"   

Once we publish the plugin names are case sensitive, so we should align before we publish, else we can't normalize it in the future.